### PR TITLE
feat: add dynamic fee indicator on Payment allocation

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

Closes https://github.com/jbx-protocol/juice-interface/issues/2915

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2934** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2934" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2934?utm_source=stack-comment).

File: src/components/Allocation/AddEditAllocationModal.tsx [hunk 1]
```diff
@@ -229,6 +254,21 @@ export const AddEditAllocationModal = ({
...
           required
+          extra={
+            showFee &&
+            amountAfterFee &&
+            ethPaymentTerminalFee && (
+              <Trans>
+                {amountAfterFee} after fee{' '}
+                <TooltipIcon
+                  tip={t`
+                      There is a ${formatFee(
```
Comment (wraeth-eth) [inline src/components/Allocation/AddEditAllocationModal.tsx:265]: Lemme know about this copy @tomquirk

File: src/components/Allocation/AddEditAllocationModal.tsx [hunk 2]
```diff
@@ -64,6 +68,27 @@ export const AddEditAllocationModal = ({
...
+    if (!ethPaymentTerminalFee || !amount?.amount) return
+    const percentageIncludedAfterFee =
+      Math.abs(100 - parseFloat(formatFee(ethPaymentTerminalFee))) / 100
+    const amountLeftover =
+      parseFloat(amount.amount) * percentageIncludedAfterFee
+    return formatCurrencyAmount({
+      amount: amountLeftover,
+      currency: allocationCurrency,
+    })
+  }, [allocationCurrency, amount?.amount, ethPaymentTerminalFee])
```
Comment (tomquirk) [inline src/components/Allocation/AddEditAllocationModal.tsx:84]: I think we have a helper fn for this already: https://github.com/jbx-protocol/juice-interface/blob/c27300806df1409d7a140ca5e108bf16624626f3/src/utils/v2v3/math.ts#L247-L254

File: src/components/Allocation/hooks/CalculateFee.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const CalculateFeeHook = () => {
+  return '00'
+}
```
Comment (tomquirk) [inline src/components/Allocation/hooks/CalculateFee.ts:3]: **question** whats the go here?

File: src/components/Allocation/AddEditAllocationModal.tsx (hunk 1 above)
Comment (tomquirk) [reply src/components/Allocation/AddEditAllocationModal.tsx:265]: @wraeth-eth lets steal all the copy from https://github.com/jbx-protocol/juice-interface/blob/c27300806df1409d7a140ca5e108bf16624626f3/src/components/v2v3/shared/FundingCycleConfigurationDrawers/DistributionSplitModal/AmountFormItem.tsx#L47-L67

**Comment by wraeth-eth** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: va1gkzPdpuku0Bz7uumv.
See details on [graphite.dev](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2934)

**Pull request merged**